eclipse-scout-core/src/table/columns/LookupColumn.ts

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+import {arrays, Column, ColumnEventMap, ColumnModel, Event, InitModelOf, LookupBox, LookupCall, LookupCallOrModel, LookupEditor, LookupResult, PropertyChangeEvent, scout, SmartField, Status, strings, TableRow} from '../../index';
+
+/**
+ * This column is a multivalued SmartColumn. If editable it opens a popup containing a {@link LookupBox} in order to select multiple values.
+ */
+export class LookupColumn<TValue> extends Column<TValue[]> implements LookupColumnModel<TValue> {
+  declare model: LookupColumnModel<TValue>;
+  declare eventMap: LookupColumnEventMap<TValue>;
+
+  lookupCall: LookupCall<TValue> = null;
+  browseHierarchy = false;
+  browseMaxRowCount = SmartField.DEFAULT_BROWSE_MAX_COUNT;
+  distinct = false;
+
+  protected override _init(model: InitModelOf<this>) {
+    super._init(model);
+
+    this._setLookupCall(this.lookupCall);
+  }
+
+  setLookupCall(lookupCall: LookupCallOrModel<TValue>) {
+    this.setProperty('lookupCall', lookupCall);
+  }
+
+  protected _setLookupCall(lookupCall: LookupCallOrModel<TValue>) {
+    let call = LookupCall.ensure(lookupCall, this.session);
+    this._setProperty('lookupCall', call);
+  }
+
+  setBrowseHierarchy(browseHierarchy: boolean) {
+    this.setProperty('browseHierarchy', browseHierarchy);
+  }
+
+  protected _setBrowseHierarchy(browseHierarchy: boolean) {
```

Comment:
the protected setters are not necessary if they only contain a call to this._setProperty -> can be removed